attestation-service/Cargo.toml

```diff
@@ -61,7 +61,9 @@ tokio.workspace = true
 tonic = { workspace = true, optional = true }
 uuid = { version = "1.1.2", features = ["v4"] }
 veraison-apiclient = { git = "https://github.com/chendave/rust-apiclient", branch = "token", optional = true }
+ear = { git = "https://github.com/chendave/rust-ear", branch = "test" }
 x509-parser = { version = "0.14.0", optional = true }
+jsonwebtoken = "8"
```

Comment:
Could you help to reorder the new two deps in dictionarial order?

Reply:
Done